cogs/custom.py

```diff
@@ -13,7 +15,7 @@ def __init__(self, client):
     # Commands
     @commands.command()
     async def hello(self, ctx):
-        await ctx.send('Hey there Buddy!')
+        await ctx.send(f'Hey there Buddy! My Creator is {constants.creator}')
```

Comment:
It seems irrelevant to mention the creator name when greeting.

Reply:
It needs improvement by a **lot**.
Well, it doesn't even need to be there if you ask me since a dummy command is of no use for the development team nor is it of use for people using the bot. We need to get some meaningful commands in.

Reply:
@Theodore-Robinson I Changed the code just now. Review That!